yogstation/code/game/objects/items/premadepapers.dm

```diff
@@ -0,0 +1,53 @@
+//yogs code begins. Paperwork pre-made edition. What a pain this was. Most of these were taken from the wiki.
+/obj/item/paper/redd
+	name = "ERROR"
+	info = "You shouldn't have this."
+
+/obj/item/paper/redd/general_request_form
+	name = "General Requests Form (Form NT-010)"
+	fields = 3
+	info = "<b>Full Name:</b> <span class=\"paper_field\"></span><br><b>Request:</b> <span class=\"paper_field\"></span><br><b>Reason for Request: </b><span class=\"paper_field\"></span><br><br><u>Sign Below:<u>"
+
+//atmospheric technician spacesuit request form
+/obj/item/paper/redd/atmos_tech_spacesuit_form
+	name = "Atmos Spacesuit Request Form (Form NT-A14)"
+	fields = 1
+	info = "I, <b><span class=\"paper_field\"></span></b>, request a <b>space suit</b> from <b>EVA</b> in the event of an atmospheric emergency requiring one.<br><b>Signed,<br>Atmospheric Technician<br></b>"
+
+//complaint form
+/obj/item/paper/redd/complaint_form
+	name = "Complaint Form (Form NT-021)"
+	fields = 4
+	info = "<b>Complainant:</b> <span class=\"paper_field\"></span><br><b>Complainant's Job: </b><span class=\"paper_field\"></span><br><b>Complainee:</b> <span class=\"paper_field\"></span><br><b>Complainee Job:</b> <span class=\"paper_field\"></span><br><b>Reason for Complaint:</b> "
+
+//cyborgization request form
+/obj/item/paper/redd/cyborg_request_form
+	name = "Cyborgization Consent Form (Form NT-203)"
+	fields = 1
+	info = "<b>I, <span class=\"paper_field\"></span> hereby consent and authorize Robotics to remove my brain and install it via an MMI into a cybernetic shell.</b><br>Signed, "
+
+//HOP access request form
+/obj/item/paper/redd/hopaccessrequestform
+	name = "HoP Access Request Form (Form NT-022)"
+	fields = 2
+	info = "<b>Name: </b><span class=\"paper_field\"></span><br><b>Accesses Requested: </b><span class=\"paper_field\"></span><br><b>Reason:</b> "
+
+//HOP job request form
+/obj/item/paper/redd/hop_job_change_form
+	name = "HoP Job Change Form (Form NT-059)"
+	fields = 3
+	info = "<b>Name: </b><span class=\"paper_field\"></span><br><b>Current Job: </b><span class=\"paper_field\"></span><br><b>Requested Job:</b> <span class=\"paper_field\"></span><br><b>Request for Change:</b> "
+
+//RD request form
+/obj/item/paper/redd/rd_form
+	name = "R&D Request Form (Form SCI-3)"
+	fields = 3
+	info = "<b>Name: </b><span class=\"paper_field\"></span><br><b>Job: </b><span class=\"paper_field\"></span><br><b>Requested Research: </b><span class=\"paper_field\"></span><br><b>Reason: </b>"
```

Comment:
This is proper useless bullshit. It takes longer to fill out the paper than to research it an chances are that if it does take longer to research it than to fill out the paper you won’t get it anyways

Reply:
Meh, it's fine to leave it in